test/index.js

```diff
@@ -51,6 +51,13 @@ describe('electron-releases', () => {
     releasesWithVersionData.length.should.be.above(154)
   })
 
+  it('includes one release with the `alpha` npm dist tag', () => {
+    const alpha = releases.filter((release) =>
+      release.npm_dist_tags.includes('alpha')
+    )
+    alpha.length.should.eq(1)
```

Comment:
Won't this number increase over time -- should this be `should.be.above(0)`?

Reply:
It's a little bit odd, but I think this test is for the `dist-tag` specifically, to make sure an alpha distribution tag exists - so the `beta` tag would always point to our latest beta release, `latest` would point to latest stable, etc, but there's only one tag. For example, beta's is here: https://github.com/electron/releases/blob/efe0422c5b866b12da98623186433f0ab244e425/test/index.js#L65